src/app/store.ts

```diff
@@ -2,5 +1,14 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers } from '@reduxjs/toolkit';
+import todosReducer from '../features/todos';
+import filterReducer from '../features/filter';
```

Comment:
Ensure that the 'filterReducer' is correctly implemented in the '../features/filter' file and is functioning as expected.